.rubocop.yml

```diff
@@ -232,7 +238,10 @@ Style/MultilineBlockChain:
 VariableInterpolation:
   Enabled: false
 
-TrailingComma:
+TrailingCommaInArguments:
```

Comment:
Actually, you should be able to set both of these trailing comma cops to true as well. (Sorry, it's been a long time since I looked at these.)